components/articles/ArticleItem.vue

```diff
@@ -21,35 +21,39 @@
           </v-layout>
         </v-card-actions>
         <v-card-title
-          class="article-item__image__content__title fill-height align-end white--text"
-          v-text="title"
-        />
+          class="article-item__image__content__title fill-height align-end white--text headline"
+          :class="compact ? 'body-1' : 'headline'"
+        >
+          <span v-text="title" />
+        </v-card-title>
       </a>
     </v-img>
-    <v-card-text class="article-item__content">
-      <loading-lines :active="!loaded" :lines="10" />
-      <div class="article-item__content__overlay"></div>
-      <p class="article-item__content__text" v-text="content"></p>
-    </v-card-text>
-    <v-card-actions>
-      <v-btn
-        :style="{ visibility: loaded ? 'visible' : 'hidden' }"
-        block
-        :href="url"
-        target="_blank"
-        text
-        color="primary"
-      >
-        Continua llegint
-      </v-btn>
-    </v-card-actions>
+    <template v-if="!compact">
+      <v-card-text class="article-item__content">
+        <loading-lines :active="!loaded" :lines="10" />
+        <div class="article-item__content__overlay"></div>
+        <p class="article-item__content__text" v-text="content"></p>
+      </v-card-text>
+      <v-card-actions>
+        <v-btn
+          :style="{ visibility: loaded ? 'visible' : 'hidden' }"
+          block
+          :href="url"
+          target="_blank"
+          text
+          color="primary"
+        >
+          Continua llegint
```

Comment:
Maybe we should add a task in order to add locales to criticat.

Reply:
Absolutely

Reply:
#16